docs/resources/kubernetes_manifest.md

```diff
@@ -22,10 +22,16 @@ Once applied, the `object` attribute reflects the state of the resource as retur
 
 - **object** (Dynamic, Optional) The resulting resource state, as returned by the API server after applying the desired state from `manifest`.
 - **wait_for** (Object, Optional) (see [below for nested schema](#nestedatt--wait_for))
+- **timeouts** (Object, Optional) (see [below for nested schema](#nestedatt--timeouts))
 
 <a id="nestedatt--wait_for"></a>
 ### Nested Schema for `wait_for`
 
 - **fields** (Map of String)
 
+<a id="nestedatt--timeouts"></a>
+### Nested Schema for `timeouts`
+
+- **delete** (Optional) Time to wait for a delete operation to succeed. Defaults to `1m`.
```

Comment:
Could we put some examples in this file? It can be hard to imagine what the resulting config looks like. It took me a while to figure it out with trial and error, even understanding what an object and map looks like.

```
  timeouts = {
    delete = "5m"
  }
```

If you view this page, you'll probably see what I mean. 

https://registry.terraform.io/providers/hashicorp/kubernetes-alpha/latest/docs/resources/kubernetes_manifest

(I didn't realize there was a separate page with examples on it until after I wrote this).